include/mbedtls/arc4.h

```diff
@@ -91,37 +94,44 @@ void mbedtls_arc4_init( mbedtls_arc4_context *ctx );
 void mbedtls_arc4_free( mbedtls_arc4_context *ctx );
 
 /**
- * \brief          ARC4 key schedule
+ * \brief          Perform an ARC4 key schedule.
  *
- * \param ctx      ARC4 context to be setup
- * \param key      the secret key
- * \param keylen   length of the key, in bytes
+ * \param ctx      The ARC4 context to be setup.
+ *                 Must not be \c NULL.
+ * \param key      The secret key buffer.
+ *                 Must be a readable buffer of length \p key Bytes.
```

Comment:
I don't quite understand this. Is it a typo and "key" is supposed to be "keylen"?

Reply:
That's a typo, thanks!

Reply:
Fixed.